rvip/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rvip)
+
+add_compile_options(-std=c++11)
+
+find_package(catkin REQUIRED COMPONENTS
+  tf
+  rospy
+  roscpp
+  roslint
+  pcl_ros
+  std_msgs
+  cv_bridge
+  sensor_msgs
+  geometry_msgs
+  image_transport
+  visualization_msgs
+)
+
+roslint_cpp()
+
+# DEPENDENCIES
+find_package(OpenGR REQUIRED PATHS /usr/local/lib/cmake)
+find_package(PCL 1.8 REQUIRED)
+
+include_directories(include ${catkin_INCLUDE_DIRS} ${PCL_INCLUDE_DIRS} ${OpenGR_INCLUDE_DIRS})
+
+link_directories(${PCL_LIBRARY_DIRS} )
```

Comment:
This should not be necessary (is actually undesirable).

Reply:
Updated. [See Commit](https://github.com/cardboardcode/rvip-1/commit/a3bccad06b6c89efcd91bd2016318d722c573516)